### ISSUE TITLE
Allow Templates for Long Definition

### ISSUE THREAD

**Issue by tsela** [Alamantus/Lexiconga#19]
Hi!

First, a Happy New Year to you!

I'm still busy with my Haotyétpi dictionary, and discovered two little things that could be useful to make word entry easier to handle:

1. It would be nice if the "locked/unlocked" status of the Word Entry Form could be saved across sessions. I personally like it unlocked, so it would be great if that was a setting that was saved with my session and would automatically be applied to the form when I log in. Right now, every time I log in I have to manually unlock the form as well, and it's an additional step I often forget to do.

2. I may have mentioned this one before (if I did, sorry about the duplication), but it would be nice to have templates available for the Explanation/Long Definition field. Just a bit of Markdown text that would be added to the field before we start typing. I like all my entries to use the same structure, and right now to do that I have to juggle with copy-paste, and that's kind of a pain. Of course, a system with multiple templates could also be practical, with a drop-down menu allowing one to choose the template one wants to apply to a particular entry.

Not sure if I'm explaining that second enhancement well, so please ask if you don't understand what I mean.

Cheers,  
Christophe

**Comment by Alamantus** [comment]
Happy new year to you too! And thanks again for your really good enhancement ideas!

As you may or may not already know, I'm (slowly) working on a full rewrite of Lexiconga using a new system (progress and updates for that are being tweeted at https://twitter.com/LexicongaDev), and one of the design changes is to just have the word list permanently unlocked because I couldn't think of any situation that a real person would ever actually want or need it locked. :P

As for templates, I really like that idea! I'll definitely add that to the settings in the new version.

**Comment by tsela** [comment]
Hi!

Thanks for your reply. I must say I am really looking forward to the new version. Any idea on the ETA? I know, "it will be ready when it is ready", but it would be nice to know what you're aiming at so I can get ready to review and test the new version as soon as it's up :).

Cheers,
Christophe

**Comment by Alamantus** [comment]
I'd like to finish before the end of Q1 this year, but I don't really have any reliable way to project an estimate since I got a new job recently. :sweat_smile: I really appreciate your willingness and eagerness to help test it, though! :smiley:

-Robbie
